Python/PRP/Message/pySoundMsg.cpp

```diff
@@ -0,0 +1,345 @@
+/* This file is part of HSPlasma.
+ *
+ * HSPlasma is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * HSPlasma is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with HSPlasma.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <PyPlasma.h>
+#include <PRP/Message/plSoundMsg.h>
+#include "pySoundMsg.h"
+#include "pyMessageWithCallbacks.h"
+
+extern "C" {
+
+static PyObject* pySoundMsg_new(PyTypeObject* type, PyObject* args, PyObject* kwds) {
+    pySoundMsg* self = (pySoundMsg*)type->tp_alloc(type, 0);
+    if (self != NULL) {
+        self->fThis = new plSoundMsg();
+        self->fPyOwned = true;
+    }
+    return (PyObject*)self;
+}
+
+static PyObject* pySoundMsg_getCmd(pySoundMsg* self, PyObject* args) {
+    int cmd;
+    if (!PyArg_ParseTuple(args, "i", &cmd)) {
+        PyErr_SetString(PyExc_TypeError, "getCmd expects an int");
+        return NULL;
+    }
+    return PyBool_FromLong(self->fThis->getCmd(cmd) ? 1 : 0);
+}
+
+static PyObject* pySoundMsg_setCmd(pySoundMsg* self, PyObject* args) {
+    int cmd;
+    int value = 1;
+    if (!PyArg_ParseTuple(args, "i|i", &cmd, &value)) {
+        PyErr_SetString(PyExc_TypeError, "setCmd expects an int and an optional int");
+        return NULL;
+    }
+    self->fThis->setCmd(cmd, value != 0);
```

Comment:
It would be handy if `plSoundMsg::setCmd` also defaulted to `value=true`, to match the Python behavior